airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -55,8 +55,8 @@
         "order": 5
       },
       "jdbc_url_params": {
-        "description": "Additional properties to pass to the JDBC URL string when connecting to the database formatted as 'key=value' pairs separated by the symbol '&'. (example: key1=value1&key2=value2&key3=value3).",
-        "title": "JDBC URL Params",
+        "description": "Additional properties to pass to the JDBC URL string when connecting to the database formatted as 'key=value' pairs separated by the symbol '&'. (example: key1=value1&key2=value2&key3=value3). Read <a href=\"https://docs.airbyte.com/integrations/sources/postgres#jdbc-url-params\">Postgres Source</a> docs for more information.",
+        "title": "JDBC URL Params (Advanced)",
```

Comment:
nonblocking: do we have a docs link explaining what "Advanced" means? This might be a little confusing to users on its own.

Reply:
We have a separate FE RFC in the works that will add ability to mark fields as advanced